ui/selectors/selectors.js

```diff
@@ -2529,6 +2529,11 @@ export function getSnapsInstallPrivacyWarningShown(state) {
   return snapsInstallPrivacyWarningShown;
 }
 
+export function getConfirmationAdvancedDetailsOpen(state) {
```

Comment:
For clarity, I think we're trying to name all new selectors `selectX` as recommended by Redux to make it explicit what are selectors and what are utils etc?

Maybe `selectShowConfirmationsAdvancedDetails`?